### ISSUE TITLE
Fine tune sorting and filter behavior on samples table

### ISSUE THREAD

**Issue by dvenprasad** [AlexsLemonade/refinebio-frontend#705]
From UE, people stumbled a little while using Filter and sorting since they did not behave as expected.

- Sorting did not really change the types of samples user expected to come on top.

- Filtering did not bring results to the top and also did not "narrow" down the number of samples on the samples table.

**Comment by dvenprasad** [comment]
Thoughts from meeting with @arielsvn:
- Exclude the additional metadata from filtered search on samples table.
- Sorting is working fine (no issues there)

Other notes: 
- Once we get the MetaSRA metadata integrated, filtering on samples table will be better. That would be a good time to pick up #710 
- We are unsure of how to integrate contents under the `Additional Metadata` column into the table filter feature. Currently, even thought the filter does search through the `Additional Metadata` column, it is a challenge to clearly indicate that users in a way that would make their life easier. So we have decided to exclude it for now. 
- Another solution that we discussed was to allow users add fields in `Additional Metadata` as column so they can easily compare samples but have not decided to implement this  because this might not be necessary after the MetaSRA integration (🤞)
